Demo.xcodeproj/project.pbxproj

```diff
@@ -910,7 +910,7 @@
 					"$(PROJECT_DIR)/iOSDemo",
 				);
 				INFOPLIST_FILE = iOSDemo/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11;
```

Comment:
Any particular reason this needs to change? Might be helpful to have the demo app still support whatever the lowest version the lib supports is.

Reply:
Not really. I just dislike `#available`. 😅